test/screens/auth/edit_account_test.dart

```diff
@@ -0,0 +1,198 @@
+import 'dart:math';
```

Comment:
I added test for edit_account as well so there should be one file called edit_account_widget_test and the other edit_account_unit_test. Please put your tests in the proper one.  the edit_account_test file must be deleted